lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -90,6 +102,30 @@
         MAX: 5
     }, null);
 
+    const _ = {
+        toUpperCase: (str) => callInstanceFunc(StringInstanceToUpperCase, str),
```

Comment:
I don't see a reason why these properties can't just be methods:

```js
const _ = {
    toUpperCase(str) { callInstanceFunc(StringInstanceToUpperCase, str) },
    toLowerCase(str) { callInstanceFunc(StringInstanceToLowerCase, str) },
    ...
}
```

No point in instantiating all these lambdas if we don't need to.

Reply:
Well, we want to return the value from them, so it would become `toUpperCase(str){ return callInstanceFunc(StringInstanceToUpperCase, str) }`, which I felt was noisier. Even after your change, do lambdas still add any/a lot of overhead?

Reply:
Agreed. Not sure about performance but it's probably the case that a regular method instead of property holding a lambda is better optimized.

---
In reply to: [148357547](https://github.com/Microsoft/ChakraCore/pull/4083#discussion_r148357547) [](ancestors = 148357547)